### ISSUE TITLE
Turn this into a runnable pipeline

### ISSUE THREAD

**Issue by adriansteffan** [peekbank/peekbank-data-import#131]
Since we are in the process of glueing together more and more of the pipleine, there could be a point made to introduce a headless mode for this import part of the pipeline (using a container - I already paved the way by introducing renv) and then have flags for either uploading the data to osf or directly moving it, for cases wherer a copy of this repo and the peekbank live on the same box

Not terribly high priority though, but it could be nice to have a single "reimport, rerun, upload and update everything" button that we can run in the future

**Comment by adriansteffan** [comment]
Nvm. renv is bad. maybe we find another solution for package management, but until then this one is stuck

**Comment by smeylan** [comment]
Is the thought that renv is necessary for specifying the versions of packages? Other automated stuff (eg GH actions) installs packages without version specifications and it seems to work reasonably well (R packages tend to be more stable / change less often than Python packages)

**Comment by mcfrank** [comment]
it's definitely best practices to do package versioning. we have had renv
work ok in various production cases (e.g., experimentology). do you think
it's impossible here?

**Comment by adriansteffan** [comment]
Renv has the habit of installing packages from source, requiring a range of dependencies installed on the system like mysql or gfortran (it also does a terrible job at actually finding installed dependencies the due to some PATH issues, e.g. https://github.com/rstudio/renv/issues/1752).

While the installation may work on some machines, unexpected errors can pop up that vary heavily depending on the host machine. During the Hackathon, we tested this with Heidi's and Martin's MacBooks. Both had these dependency issues, so we decided to postpone versioning for now.

I think the 3 options to get version control running are:

1. figure out how to get renv to install packages from binaries (type = "binary" is ignored, at least for dependencies of packages)

2. Restructure peekbank to not require the troublemaker packages (mainly: move the database interface out of peekbankr and get the data via REST, that would require introducing a backend layer between client and database though) - but that seems ugly as it restricts the packages we can use overall and introduces a good chunk of extra work
 
3. Look for another versioning solution that does not have the same issues as renv
